src/libopenrave/kinbodygeometry.cpp

```diff
@@ -162,8 +162,63 @@ void AppendBoxTriangulation(const Vector& pos, const Vector& ex, TriMesh& tri)
     tri.indices.insert(tri.indices.end(), &indices[0], &indices[nindices]);
 }
 
+static void AppendConeTriangulation(const Vector& pos, const dReal rad, dReal halfHeight, const uint numFaces, TriMesh& tri, const bool upsideDown = false)
```

Comment:
@superfashi have you visualized this to confirm it creates triangles in the correct clockwise order? 
also, let's call it `radius`.

Reply:
Yes I have tested visually. Name changed.